terracotta-kit/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>terracotta-core-root</artifactId>
+        <groupId>org.terracotta.internal</groupId>
+        <version>5.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>terracotta-kit</artifactId>
+    <packaging>pom</packaging>
+    <name>Terracotta Kit</name>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.terracotta.internal</groupId>
+            <artifactId>terracotta</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.terracotta</groupId>
+                <artifactId>maven-forge-plugin</artifactId>
```

Comment:
No idea why this plugin requires the artifact to be installed.
If replaced with

```
<groupId>org.apache.maven.plugins</groupId>
<artifactId>maven-dependency-plugin</artifactId>
<version>2.10</version>
```
and after adapting the configuration - that is removing everything but the output dir which becomes `outputDirectory`, it builds fine with just the verify task.